src/main.rs

```diff
@@ -140,7 +140,7 @@ fn main() -> Result<()> {
     #[cfg(windows)]
     if config.editor.is_some() {
         unsafe {
-            bindings::Windows::Win32::SystemServices::FreeConsole();
+           crate::cargo_temp_bindings::Windows::Win32::SystemServices::FreeConsole();
```

Comment:
```suggestion
           cargo_temp_bindings::Windows::Win32::SystemServices::FreeConsole();
```

cargo_temp_bindings is not in your crate. It's its own crate